src/com/iandonaldson/data/FilmDaoImpl.java

```diff
@@ -115,13 +95,15 @@ public FilmDaoImpl() {
 	}
 
 	@Override
-	public Film getFilm(int Id) {
+	public Film getFilm(int ID) {
 		Film film = new Film();
-		
-		Connection conn = ConnectionFactory.getConnection();
+		ActorDaoImpl actorDaoImpl = new ActorDaoImpl();
 		try {
-			Statement st = conn.createStatement();
-			ResultSet rs = st.executeQuery("Select * from sakila.film where film_id = " + Id);
+			Connection conn = ConnectionFactory.getConnection();
+			
+			PreparedStatement ps = conn.prepareStatement("Select * from sakila.film where film_id = ?");
```

Comment:
I noticed some inconsistencies with your selects. Sometimes you include the database before the table and sometimes you don't. It would be good coding practice if you either did or didn't. I would suggest not to allow for portability.